### PR TITLE
openpsi cartpole component

### PR DESCRIPTION
**Pull request by amebel**

### CODE REVIEW

File: opencog-gym.py [hunk 1]
```diff
@@ -19,7 +19,7 @@
 
 # OpenAI Gym
 import gym
-env = gym.make('CartPole-v0')
+env = gym.make('CartPole-v1')
```
Comment (ngeiswei) [inline opencog-gym.py:22]: @AmeBel, just curious why did you use `v1` instead of `v0`?

**Review by ngeiswei — COMMENTED** [review]
@AmeBel the code looks good however it doesn't work for me, no warning, nothing, the program just returns after a few ms. Any idea?

(EDIT: just needs to run `main()` once in the python interpreter, duh).

File: opencog-gym.py (hunk 1 above)
Comment (amebel) [reply opencog-gym.py:22]: It is the current version, and according to the pull-request that introduced it is more challenging https://github.com/openai/gym/pull/260
For the the difference, see [here](https://github.com/openai/gym/blob/bf7e44f680fa237a3c3ae127eb096f0a9daa62a9/gym/envs/__init__.py#L53-L65 )

**Comment by ngeiswei** [comment]
Thanks @AmeBel !

**Pull request merged**